vignettes/logistic-metadata-example.Rmd

```diff
@@ -902,18 +1159,47 @@ all, but if you do, you have to document the "physical" file format it is in
 
 ```{r}
 ## check base EML
-eml_validate(my_eml)
+emld::eml_validate(my_eml)
 
 ## check that the EML is also a valid EFI forecast
 EFIstandards::forecast_validator(my_eml)
 ```
 
 ##### Python
 
-(Under construction)
+There is no Python EML validation utility.
+Therefore, we will have to use the R utility here.
+First, we write to disk.
 
-#### {-}
+```{python write-eml}
+with open("forecast-eml-python.xml", "w") as f:
+    f.write(ET.tostring(my_eml, encoding="unicode"))
+```
+
+Next, we have two options:
+One is to use the Python standard library to spawn a separate R process that runs the validator.
+
+```{python r-subprocess, eval=FALSE}
+import subprocess
+result = subprocess.run([
+    "Rscript",
+    "-e", 'emld::eml_validate("forecast-eml-python.xml")',
+    "-e", 'EFIstandards::forecast_validator("forecast-eml-python.xml")'
+], capture_output=True)
+```
+
+Alternatively, we can use the `rpy2` module to run R directly from within Python (analogous to R's `reticulate`).
+
+```{python r-rpy2, eval=FALSE}
+from rpy2.robjects.packages import importr
+emld = importr("emld")
+EFIstandards = importr("EFIstandards")
+
+emld.eml_validate("forecast-eml-python.xml")
+EFIstandards.forecast_validator("forecast-eml-python.xml")
+```
```

Comment:
@mdietze AFAIK, this is all it takes to call the EML and EFI validators from Python. Is there anything else you want me to do with this? Write a script? Or can we just point to this code when describing our validator, and/or copy-paste basically this snippet in our documentation?